.github/workflows/scripts/ti_build/entry.py

```diff
@@ -33,11 +33,11 @@ def build_wheel(python: Command, pip: Command) -> None:
     cmake_args.writeback()
     u = platform.uname()
     if (u.system, u.machine) == ("Linux", "x86_64"):
-        extra.extend(["-p", "manylinux_2_27_x86_64"])
+        extra.extend(["-p", "manylinux_2_34_x86_64"])
     elif (u.system, u.machine) in (("Linux", "arm64"), ("Linux", "aarch64")):
-        extra.extend(["-p", "manylinux_2_27_aarch64"])
+        extra.extend(["-p", "manylinux_2_34_aarch64"])
```

Comment:
We might do this. Note that this doesnt affect the build process. And at test time, this *reduces* the possible installation targets. ie if this is set to 2 27, we can install on: 2 27, 2 28, 2 34. If we set it to 2 34, it can only be installed on: 2 34.

Reply:
(this basically affects the filename fo the wheel, which is used by pip to decide if it's ok to try to install the wheel)